packages/fxa-admin-server/src/test/lib/db/models/helpers.ts

```diff
@@ -45,12 +45,19 @@ export function randomEmailBounce(email: string) {
   };
 }
 
-interface AccountIsh {
+export interface AccountIsh {
   uid: string;
   email: string;
   normalizedEmail: string;
 }
 
+export interface BounceIsh {
+  bounceSubType: number;
+  bounceType: number;
+  createdAt: number;
+  email: string;
+}
+
```

Comment:
Could `Pick` or `Omit` be helpful here to make the relationships between these `*ish` types and the "real" types easier to understand?  E.g. `BounceIsh = Pick<EmailBounce, 'bounceSubType' | 'bounceType' | 'createAt' | 'email'>`.  The types of the properties will also stay n*sync.